stories/Appointment.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import { Story, Meta } from '@storybook/react'
+
+import { Appointment as AppointmentComponent, AppointmentProps } from '../components/Appointment'
+
+export default {
+  title: 'Component/Atoms/Appointment',
+  component: AppointmentComponent,
+  argTypes: {
+    date: {
+      description: 'Please ignore the time picker',
+      control: {
+        type: 'date',
+      },
+    },
+    start: {
+      name: 'start time',
+      table: {
+        type: {
+          summary: 'number',
+        },
+      },
+      control: {
+        type: 'number',
+        min: 1,
+        max: 12,
+      },
+    },
+    end: {
+      name: 'end time',
+      table: {
+        type: {
+          summary: 'number',
+        },
+      },
+      control: {
+        type: 'number',
+        min: 1,
+        max: 12,
+      },
+    },
+    timeSlot: {
+      table: {
+        disable: true,
+      },
+    },
+  },
```

Comment:
Note: This hides the `timeSlot` argument from the Storybook Controls. Let me know if you disagree with this approach.